Makefile

```diff
@@ -19,16 +19,29 @@ format:  ## Format shell scripts
 .PHONY: check format
 
 # --- Release -------------------------------------------------------------------
-NEXT_TAG := $(shell { git tag --list --merged HEAD --sort=-v:refname; echo v0.0.0; } | grep -E "^v?[0-9]+.[0-9]+.[0-9]+$$" | head -n1 | awk -F . '{ print $$1 "." $$2 "." $$3 + 1 }')
-MAJOR_RELEASE := $(firstword $(subst ., ,$(NEXT_TAG)))
 
-release:  ## Tag release
-	git tag --annotate --message "Release $(NEXT_TAG)" $(NEXT_TAG)
-	git push origin $(NEXT_TAG)
+## Tag and release binaries for different OS on GitHub release
+release: nexttag ## Release and tag release
```

Comment:
This has both the old and the new help message format - the line before and same line suffix. This repo only does the latter right now

Reply:
ok, addressed, thanks.